xborders

```diff
@@ -128,14 +128,22 @@ def get_args():
 
     return
 
-
+# Every call of get_active_window() instantiates a new
+# window object, in this way, the active_window object
+# is destroyed in this scope but only the focused
+# window remains.
 def get_win(wnck_screen):
-	active_window = wnck_screen.get_active_window()
-	if (active_window):
-		x, y, width, height = active_window.get_geometry()
-		return ((x, y), (width, height))
-	else:
-		return None
+    global BORDERED_WINDOW
```

Comment:
I don't understand the logic here, you store the focused window in the BORDERED_WINDOW global variable and that's it. Every time when you call get_win it will still call get_active_window which will construct the window object like before.

Reply:
Woke up thinking the same,
Do I revert it or do you think having global access to the active window without calling ```get_win()``` would be useful for other features?

Reply:
I think the idea is okay, but it would be better to have some time based caching mechanism for it.
Something like this: https://github.com/deter0/xborder/blob/main/xborders#L144

Reply:
And maybe it would be easier to move the `get_win` method inside the class so we could eliminate the usage of global variables.